src/precice/impl/DataContext.hpp

```diff
@@ -44,6 +44,20 @@ class DataContext {
    */
   int getDataDimensions() const;
 
+  /**
+   * @brief Get the spatial dimensions of _providedData.
+   *
+   * @return int Spatial dimensions of _providedData.
+   */
+  int getSpatialDimensions() const;
```

Comment:
in the API, we now call this `getMeshDimensions`. Does this make sense here as well?

Reply:
Puh... I'm a bit unsure. This has quite some implications that don't really belong here in my opinion. There is, for example, also a `Data::getMeshDimensions()` and `Data::_spatialDimensions`. I would rather rename this is another PR.